ocw/lib/db.py

```diff
@@ -34,18 +34,20 @@ def sync_csp_to_local_db(pc_instances, provider, namespace):
         if i.vault_namespace != namespace:
             raise ValueError('Instance {} does not belong to {}'.format(i, namespace))
 
-        logger.debug("Update/Create instance %s:%s @ %s", provider, i.instance_id, i.region)
-        if Instance.objects.filter(provider=i.provider, instance_id=i.instance_id).exists():
-            o = Instance.objects.get(provider=i.provider, instance_id=i.instance_id)
+        if Instance.objects.filter(provider=i.provider, instance_id=i.instance_id, vault_namespace=namespace).exists():
+            logger.debug("[%s] Update instance %s:%s", namespace, provider, i.instance_id)
+            o = Instance.objects.get(provider=i.provider, instance_id=i.instance_id, vault_namespace=namespace)
             if o.region != i.region:
-                logger.info("Instance %s:%s changed region from %s to %s",
-                            provider, i.instance_id, o.region, i.region)
+                logger.info("[%s] Instance %s:%s changed region from %s to %s",
+                            namespace, provider, i.instance_id, o.region, i.region)
                 o.region = i.region
             if o.state == StateChoice.DELETED:
-                logger.error("Update already DELETED instance %s:%s\n\t%s", provider, i.instance_id, i.csp_info)
+                logger.error("[%s] Update already DELETED instance %s:%s %s",
+                             namespace, provider, i.instance_id, i.all_time_fields())
             if o.state != StateChoice.DELETING:
                 o.state = StateChoice.ACTIVE
```

Comment:
```suggestion
                logger.error("[%s] Update and reactivating DELETING instance %s:%s %s",
                             namespace, provider, i.instance_id, i.all_time_fields())
                o.state = StateChoice.ACTIVE
```

Reply:
Keep in mind that message coming just after this for same instance `logger.debug("[%s] Update instance %s:%s", namespace, provider, i.instance_id)` . While I agree with idea to mention instance activation I would suggest different message : 
`logger.error("[%s] %s:%s instance which still exists has DELETED state in DB. Reactivating %s",
                             namespace, provider, i.instance_id, i.all_time_fields())`